.github/workflows/run_main_benchmarks.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd $BRANCH_NAME/
```

Comment:
Why do we need this `cd $BRANCH_NAME/` command in this script and also `run_pr_benchmarks.sh`?

Reply:
I just kept this for the sake of consistency across the two files. I am running the PR benchmark on the `$BRANCH_NAME/pr/NullAway` folder _(the whole `$BRANCH_NAME` directory was set up to avoid errors due to any residual files from other PRs)_, and I am navigating back to the root directory after that's done. I can omit the` cd $BRANCH_NAME` from `main`, but that would mean that I will have to go up two directories to the `$BRANCH_NAME` folder in the step where I run the gcloud ssh command. Happy to make those changes if they are more suitable @msridhar. Thank you!

Reply:
Ok, it's fine to keep, was just curious.